jest.config.base.ts

```diff
@@ -2,27 +2,11 @@ import fs from "fs";
 import path from "path";
 
 import type { Config } from "@jest/types";
-// TODO: Use `pathsToModuleNameMapper` instead.
-// @ts-ignore
-import tsconfigPaths from "tsconfig-paths-jest";
+import { pathsToModuleNameMapper } from "ts-jest";
```

Comment:
NOTE: This caused errors when running `jest` from the root, rather an individual `packages` folder.